src/CoreWCF.Primitives/src/CoreWCF/Security/SecuritySessionSecurityTokenAuthenticator.cs

```diff
@@ -1106,14 +1106,23 @@ public SecuritySessionHost(SecuritySessionSecurityTokenAuthenticator authenticat
             }
 
             internal ChannelDispatcher InitializeRuntime(SecurityServiceDispatcher securityDispatcher)
+            {
+                if (securityDispatcher.AcceptorChannelType.Equals(typeof(IReplyChannel)))
+                    return InitializeRuntime<IReplyChannel>(securityDispatcher);
```

Comment:
Curly braces around if blocks. Also blank lines after if blocks too so make sure there's a blank line before the throw statement.